lib/improver/tests/utilities/test_mathematical_operations.py

```diff
@@ -359,7 +410,7 @@ def test_start_point_negative(self):
             result.coord("height").points, np.array([5.]))
         self.assertArrayAlmostEqual(result.data, expected)
 
-    def test_end_point_positive(self):
+    def test_end_point_positive_direction(self):
```

Comment:
This doc-string says negative direction, but the title says positive (it appears that it is positive, so the doc-string needs fixing).

Reply:
Amended docstring.